recipes/daemoniker/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "daemoniker" %}
+{% set version = "0.2.3" %}
+{% set hash = "d9b62c8d75227b3da88c0ca22b82e19d88d32dade4ffa838427235e0cdd5e578" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - daemoniker
+
+about:
+  home: https://github.com/Muterra/py_daemoniker
+  license: Unlicense
+  license_family: PUBLIC-DOMAIN
+  license_file: UNLICENSE.txt
+  summary: 'Daemoniker provides a cross-platform Python API for running and signaling daemonized Python code'
+  description: |
+    Daemoniker also provides several utility tools for the resulting daemons.
+    In particular, it includes cross-platform signaling capability for the created daemons.
+  doc_url: https://daemoniker.readthedocs.io/en/latest/
+  dev_url: https://github.com/Muterra/py_daemoniker
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - nerohmot
```

Comment:
Hi @nerohmot , can we have a confirmation from you that you would like to maintain this recipe as well please?

Reply:
Actually no need to co-maintain this @nerohmot, I can handle it :)